dotnetv3/AutoScaling/scenarios/AutoScale_Basics/AutoScale_Basics/AutoScaleBasics.cs

```diff
@@ -0,0 +1,247 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+  Before running this SDK for .NET (v3) code example, set up your development
+  environment, including your credentials.
+
+  For more information, see the following documentation:
+
+  https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-setup.html
+
+  This example creates an Amazon Elastic Compute Cloud (Amazon EC2) launch
+  template. You can also use the Amazon EC2 console. For more information, see
+  the following:
+
+  https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-launch-templates.html#create-launch-template
+
+ This code example performs the following operations:
+  1. Creates an Amazon EC2 Launch Template
+  2. Creates an AWS Auto Scaling group.
+  3. Shows the details of the new AWS Auto Scaling group to show that only
+     one instance was created.
+  4. Enables metrics collection.
+  5. Updates the AWS Auto Scaling group to increase the max size to three.
```

Comment:
5. Updates the Auto Scaling group to increase the maximum size to three.

Reply:
fixed